ui/src/tool-panel/location-track/splitting/location-track-split.tsx

```diff
@@ -18,18 +17,34 @@ import {
 import {
     FirstSplitTargetCandidate,
     SplitTargetCandidate,
+    SwitchOnLocationTrack,
 } from 'tool-panel/location-track/split-store';
-import { MAP_POINT_NEAR_BBOX_OFFSET } from 'map/map-utils';
+import {
+    calculateBoundingBoxToShowAroundLocation,
+    MAP_POINT_NEAR_BBOX_OFFSET,
+} from 'map/map-utils';
 import NavigableTrackMeter from 'geoviite-design-lib/track-meter/navigable-track-meter';
 import { isEqualIgnoreCase } from 'utils/string-utils';
-import { getOperation } from './split-utils';
+import {
+    END_SWITCH_NOT_MATCHING_ERROR,
+    getOperation,
+    START_SWITCH_NOT_MATCHING_ERROR,
+} from './split-utils';
+import { IconColor, Icons, IconSize } from 'vayla-design-lib/icon/Icon';
+import { BoundingBox, Point } from 'model/geometry';
 
-type EndpointProps = {
+type CommonProps = {
```

Comment:
Nimenä vähän geneerinen, mutta toisaalta eipä tämä täältä mihinkään uloskaan exporttaudu. Tälle voisi koittaa keksiä jonkun vähän paremman nimen, mutta myönnettäköön että en kyllä itsekään keksi ihan suoraan mikä olisi hyvä englanninkielinen termi tälle.

Reply:
Selvästikin split 😄 

Mitäs näitä muita olis, esim. threshold/junction/juncture/coupling ei meillä ainakaan muualla käytössä muistaakseni käytössä. Osalla tosin ratamaailmassa jo muuta käyttöä